pkg/indexmanagement/scripts.go

```diff
@@ -35,7 +35,13 @@ fi
 CMD=$(cat <<END
 from __future__ import print_function
 import json,sys
-r=json.load(sys.stdin)
+from io import StringIO
+indexStr = sys.stdin.readline().rstrip()
```

Comment:
The new code only works if the JSON string is always a single line.
The old code would read multi-line JSON.
Is that a safe assumption?